### PR TITLE
added user management section

### PR DESCRIPTION
**Pull request by MaxWardle**

### CODE REVIEW

**Review by wrnu — APPROVED** [review]

File: docs/user-management.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# BC Developer Registry - User Management Documentation
+Last updated: **March 12, 2024**
+
+Welcome to the BC Developer Registry documentation. This guide provides detailed information about the new user interface changes and the enhanced user management feature available at [BC Gov's Product Registry](https://registry.developer.gov.bc.ca/). With the latest update, users can now manage AWS permissions directly from the website, enhancing the overall user experience and streamlining project management.
```
[the remark this answers is not in the document]
Comment (Pilargit12) [reply docs/user-management.md:4]: Calling it multiple names can cause confusion to readers, make sure you are using the name of the product itself as called by the Private and Public cloud

File: docs/user-management.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+## New User Interface
+
+The BC Developer Registry has undergone significant UI improvements to provide a more intuitive and user-friendly experience. You can now easily navigate through the website, access different sections, and manage your projects and users more efficiently.
+
+## User Management Feature
+
+One of the key updates is the introduction of a comprehensive user management feature. This feature allows product owners and technical leads to manage AWS permissions for users associated with their product. Users can be assigned to one or more of the following roles:
+
+1. **Viewers**
+2. **Billing Viewers**
```
Comment (Pilargit12) [inline docs/user-management.md:15]:
```suggestion
2. **Billing viewers**
```

File: docs/user-management.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+3. **Developers**
+4. **Admins**
+5. **Security Auditors**
```
Comment (Pilargit12) [inline docs/user-management.md:18]:
```suggestion
5. **Security auditors**
```

File: docs/user-management.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+![Registry Roles](images/user-management/registry_roles.png)
+### Prerequisites
+
+- Users must have a valid IDIR to be added to these roles.
+- Only product owners or technical leads can view and manage users in the website.
+
+### Role-Based Access
```
Comment (Pilargit12) [inline docs/user-management.md:26]:
```suggestion
### Role-based access
```

File: docs/user-management.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Admins
+
+- **Policy**: `arn:aws:iam::aws:policy/AdministratorAccess`
+- Admins have full access to all four AWS accounts within the product: dev, test, prod, and tools.
+
+#### Developers
+
+- **Policy**: `arn:aws:iam::aws:policy/AdministratorAccess`
+- Developers have admin access to dev, test, and tools accounts. To gain admin access to the prod account, they must have the Admin role.
```
Comment (Pilargit12) [inline docs/user-management.md:38]:
```suggestion
- Developers have admin access to dev, test, and tools accounts. To gain admin access to the prod account, they must have the admin role.
```

File: docs/user-management.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Viewers
+
+- **Policy**: `arn:aws:iam::aws:policy/ReadOnlyAccess`
+- Viewers have read-only access to the AWS accounts.
+
+#### Security Auditors
```
Comment (Pilargit12) [inline docs/user-management.md:45]:
```suggestion
#### Security auditors
```

File: docs/user-management.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+- **Policy**: `arn:aws:iam::aws:policy/SecurityAudit`
+- Security Auditors have access to security configuration and logs to perform audit tasks.
```
Comment (Pilargit12) [inline docs/user-management.md:48]:
```suggestion
- Security auditors have access to security configuration and logs to perform audit tasks.
```

File: docs/user-management.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+
+#### Billing Viewers
```
Comment (Pilargit12) [inline docs/user-management.md:50]:
```suggestion
#### Billing viewers
```

File: docs/user-management.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+
+- **Custom Policy**: Provides access to AWS Budgets and Cost Explorer.
```
Comment (Pilargit12) [inline docs/user-management.md:52]:
```suggestion
- **Custom policy**: Provides access to AWS budgets and cost explorer.
```

File: docs/user-management.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+            "Effect": "Allow",
+            "Resource": "*",
+            "Sid": "AllowBudgets"
+        }
+    ],
+    "Version": "2012-10-17"
+}
+```
+
+## Accessing the Public Cloud Landing Zone
```
Comment (Pilargit12) [inline docs/user-management.md:74]:
```suggestion
## Accessing the Public Cloud AWS Landing Zone
```

Comment (Pilargit12) [reply docs/user-management.md:74]: Please be careful with calling this also different names across

File: docs/user-management.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+- Once a user is added to a role, they can access the product (AWS accounts) from the [Public Cloud Landing Zone.](https://login.nimbus.cloud.gov.bc.ca/) If users have multiple roles, they can choose any of those roles when logging in to the account.
+![Landing Zone](images/user-management/landing_zone.png)
+
+## AWS Managed Policies Overview
```
Comment (Pilargit12) [inline docs/user-management.md:78]:
```suggestion
## AWS managed policies overview
```

File: docs/user-management.md [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+- AWS managed policies are predefined policies that provide permissions for many common use cases. These policies are maintained by AWS and provide a set of permissions that align with common job functions in the AWS environment. By using AWS managed policies, you can ensure that the permissions are set up correctly and updated by AWS as necessary.
```
Comment (Pilargit12) [inline docs/user-management.md:79]:
```suggestion
- AWS managed policies are predefined policies that provide permissions for many common use cases. These policies are maintained by AWS and provide a set of permissions that align with common job functions in the AWS environment. By using AWS managed policies, you can ensure that the permissions are set up correctly and updated by AWS as necessary.

```

File: docs/user-management.md [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+    - AdministratorAccess: Provides full access to AWS services and resources.
+    - ReadOnlyAccess: Provides read-only access to AWS services and resources.
+    - SecurityAudit: Provides permissions to view configuration data for security auditing.
+
+# Conclusion
+- The new user management feature in the BC Developer Registry enhances the ability of product owners and technical leads to manage AWS permissions efficiently. By understanding the roles and their associated policies, you can ensure that users have the appropriate access to your AWS resources.
```
Comment (Pilargit12) [inline docs/user-management.md:85]:
```suggestion
- The new user management feature in the BC Gov's Product Registry enhances the ability of product owners and technical leads to manage AWS permissions efficiently. By understanding the roles and their associated policies, you can ensure that users have the appropriate access to your AWS resources.
```

File: mkdocs.yml [hunk 14]
```diff
@@ -7,7 +7,8 @@ nav:
   - Provision a project set in AWS: provision-a-project-set.md
 - Design, Build and Deploy an Application:
   - Requirements for building your application: requirements-for-building-your-application.md
-  - Deploy an app to BC Gov's AWS Landing Zone: deploy-an-app-to-the-aws-landing-zone
+  - User Management: user-management.md
```
Comment (Pilargit12) [inline mkdocs.yml:10]:
```suggestion
  - User management: user-management.md
```

File: docs/user-management.md [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+            "Sid": "AllowBudgets"
+        }
+    ],
+    "Version": "2012-10-17"
+}
+```
+
+## Accessing the Public Cloud Landing Zone
+- Once a user is added to a role, they can access the product (AWS accounts) from the [Public Cloud Landing Zone.](https://login.nimbus.cloud.gov.bc.ca/) If users have multiple roles, they can choose any of those roles when logging in to the account.
+![Landing Zone](images/user-management/landing_zone.png)
```
Comment (Pilargit12) [inline docs/user-management.md:76]: Please add an explanation of this image for accessibility purposes, describe either in list form what this images describes and provide an alt.text

https://www2.gov.bc.ca/gov/content/home/accessible-government/toolkit/accessible-digital-content/graphics

File: docs/user-management.md [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+
+## AWS Managed Policies Overview
+- AWS managed policies are predefined policies that provide permissions for many common use cases. These policies are maintained by AWS and provide a set of permissions that align with common job functions in the AWS environment. By using AWS managed policies, you can ensure that the permissions are set up correctly and updated by AWS as necessary.
+    - AdministratorAccess: Provides full access to AWS services and resources.
+    - ReadOnlyAccess: Provides read-only access to AWS services and resources.
+    - SecurityAudit: Provides permissions to view configuration data for security auditing.
```
Comment (Pilargit12) [inline docs/user-management.md:82]:
```suggestion
    - SecurityAudit: Provides permissions to view configuration data for security auditing
```

File: docs/user-management.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+}
+```
+
+## Accessing the Public Cloud Landing Zone
+- Once a user is added to a role, they can access the product (AWS accounts) from the [Public Cloud Landing Zone.](https://login.nimbus.cloud.gov.bc.ca/) If users have multiple roles, they can choose any of those roles when logging in to the account.
+![Landing Zone](images/user-management/landing_zone.png)
+
+## AWS Managed Policies Overview
+- AWS managed policies are predefined policies that provide permissions for many common use cases. These policies are maintained by AWS and provide a set of permissions that align with common job functions in the AWS environment. By using AWS managed policies, you can ensure that the permissions are set up correctly and updated by AWS as necessary.
+    - AdministratorAccess: Provides full access to AWS services and resources.
```
Comment (Pilargit12) [inline docs/user-management.md:80]:
```suggestion
    - AdministratorAccess: Provides full access to AWS services and resources
```

File: docs/user-management.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+    - ReadOnlyAccess: Provides read-only access to AWS services and resources.
```
Comment (Pilargit12) [inline docs/user-management.md:81]:
```suggestion
    - ReadOnlyAccess: Provides read-only access to AWS services and resources
```

File: docs/user-management.md [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+
+One of the key updates is the introduction of a comprehensive user management feature. This feature allows product owners and technical leads to manage AWS permissions for users associated with their product. Users can be assigned to one or more of the following roles:
+
+1. **Viewers**
+2. **Billing Viewers**
+3. **Developers**
+4. **Admins**
+5. **Security Auditors**
+
+![Registry Roles](images/user-management/registry_roles.png)
```
[the remark this answers is not in the document]
Comment (MaxWardle) [reply docs/user-management.md]: See commit 78fbd767ff9a51f60ab6523a40c4ab7ec57e1be8

**Pull request merged**